sos/report/plugins/storcli64.py

```diff
@@ -0,0 +1,46 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+import os
+
+
+class StorCLI64(Plugin, IndependentPlugin):
+
+    short_desc = 'LSI MegaRAID devices'
+
+    plugin_name = 'storcli64'
+    profiles = ('system', 'storage', 'hardware',)
+    packages = ('storcli',)
+
+    def setup(self):
+        os.environ['PATH'] += os.pathsep + "/opt/MegaRAID/storcli"
```

Comment:
Nack to `PATH` modifications - same as in #2610

Reply:
Looking forward for your recommendation to fix the path.